packages/abi-coder/src/abi-coder.ts

```diff
@@ -204,10 +114,10 @@ export abstract class AbiCoder {
       return new TupleCoder(coders);
     }
 
-    return logger.throwArgumentError('Coder not found', 'type', { abiType, abi });
+    return logger.throwArgumentError('Coder not found', 'abiType', { abiType, abi });
   }
 
-  private static getCoders(components: readonly JsonAbiArgument[], abi: JsonAbi) {
+  private static getCoders(abi: JsonAbi, components: readonly JsonAbiArgument[]) {
```

Comment:
I played around with the signature of this method a lot because I was trying out different designs that affected it. In the end I went back to this one, but changed the order... I thought it was already like this, no specific intent. Anyways, it makes sense to have it like this as you can't have coders for components without the abi, thus making the abi the "parent", "more important".